package.json

```diff
@@ -35,11 +35,11 @@
 		"any-date-parser": "1.5.3"
 	},
 	"peerDependencies": {
-		"luxon": "^1.26.0"
+		"luxon": ">=1.28.0"
```

Comment:
support luxon 1 through 3 since as of today, luxon is still making releases to all three versions.